### DISCUSSION TITLE
📎 Add `allowReturnOutsideFunction` option to the parser

### DISCUSSION THREAD

**Discussion by Boshen** [rome/tools#2965] [Suggestions]
### Description

From babel
> allowReturnOutsideFunction: By default, a return statement at the top level raises an error. Set this to true to accept such code.

Node scripts and bundlers all support this.

An example found in the wild: https://github.com/microsoft/vscode/blob/main/scripts/update-xterm.js#L24

**Comment by ematipico** [comment]
@MichaReiser @xunilrj what do you think? Is it a configuration that we really need? Is it something that we can deduce from the environment?
